### DISCUSSION TITLE
Toolbox-Symlink

### DISCUSSION THREAD

**Discussion by Doris1337** [diStyApps/seait#29] [Q&A]
Hi, 

I have a bit of an issue with creating Symlink, I find it fantastic the idea but I think I'm doing something wrong. Would love if the creater ,coder, developer or how ever he she it identifies ... whatever (little woke joke)  
could crarify or create a little tutorial on how properly to do it. I don't have it clear what would be the source (I suppose folder containing models) and target (folders containing apps) but I end up having folder going into ROLLback. Still love the idea of Symlink and find it very necesary. Thanks still for creating the easy installer and keeping it updated. 

awesome work

**Comment by diStyApps** [comment]
Thank you and sorry for the inconvenience.
It is a permissions issue mainly, try to run as Run as Administrator, but it might not be enough.
Here is an explanation how to set it up:
https://github.com/diStyApps/seait/releases/tag/0.1.4
in next version it will be easier, video tutorial will be out soon.
